blog/guide.ftd

```diff
@@ -0,0 +1,300 @@
+-- import: fastn.com/blog/authors
+-- import: fastn-community.github.io/doc-site/common
+-- import: fastn-community.github.io/bling/note
+-- import: fastn.com/utils
+
+
+-- common.post-meta meta: Building Your Personal Website with fastn: A Step-by-Step Guide
+published-on: November 30, 2023
+post-url: /blog/guide/
+author: $authors.nandini
+
+Today, I’m excited to share my journey of creating my personal website using 
+`fastn`. I'm a freelance content writer, and [my existing website](https://www.canva.com/design/DAFbReZ9W-8/UC3KC6uGXfx_vRCnohjdFA/view?website#2), 
+built on Canva, felt more like scrolling through a series of slides than a 
+professional website. It lacked vital elements like a navigation bar, blog page, 
+and a contact form. 
+
+
+-- ds.blog-page:
+meta: $meta
+
+-- ds.h1: The Limitations of Canva and Notion
+
+Worse yet, my website on Canva took ages to load, forcing me to host 
+[my portfolio](https://nandhinidevi.notion.site/Hi-I-m-Nandhini-9f393b0846ad472c95529d94fb03d4b8) 
+separately on Notion, which, despite its functionality, couldn’t match my vision 
+due to limited design elements. 
+
+Honestly, my portfolio there looks more like school notes than a professional 
+showcase. If you've used Notion, you know what I mean—it’s not the vibe I’d want 
+to showcase to potential clients.
+
+Then came fastn! When I [first started exploring fastn](https://fastn.com/writer/) 
+I was eager to try it out for my website - one that has everything I'd like to 
+showcase to potential clients: details about me, my clients, projects, services, 
+blogs, and a user-friendly contact form. My goal was to achieve a clean, simple, 
+yet visually appealing design and layout.
+
+-- ds.h1: Finding the perfect template
+
+Browsing through the templates on the [featured components page](http://fastn.com/featured), 
+I found exactly what I needed. This single page [fastn template](https://fastn.com/featured/portfolios/portfolio/) 
+under the Portfolio/Personal Site category ticked all the boxes. 
+
+Its structure has a single-page site with a vertical navigation bar, allowing 
+users to swiftly navigate to specific sections. This feature is a must for a 
+lengthy landing page. You can easily view the code by clicking on the GitHub 
+icon if you’re logged into your GitHub account. 
+
+Discover my step-by-step process in building this website with fastn in the 
+video below:
+
+/-- ds.youtube:
+\v: 
+
+-- ds.h2: A walkthrough on the files and folders in this template
+
+-- ds.h3: [**`fastn.ftd`**](https://github.com/fastn-community/saurabh-portfolio/blob/main/FASTN.ftd) file
+
+This file contains all the website imports and its sitemap, outlining sections 
+and their corresponding URLs.
+
+-- ds.code:
+lang: ftd 
+
+\-- fastn.sitemap:
+
+\# Home: index.html
```

Comment:
You do not need to "escape" (this is what we call when we add `\` before `--` above) heading here.

```suggestion
# Home: index.html
```